### PR TITLE
Add 'news' section to the website for posting updates

### LINKED ISSUE #465: Create blog section on website
This is a low friction location to put out ideas, examples, use-cases, planning notes, developer status, etc.

Include https://docs.google.com/document/d/1jit6mBSyEW-p027sx2dK7ySQaEa-lkKxmkcFyRXHOHo/edit?usp=sharing as the first post.

### PR DESCRIPTION
**Pull request by MikaelSmith**
Seed it with an intro to Wash post.

Resolves #465.

Signed-off-by: Michael Smith <michael.smith@puppet.com>

### CODE REVIEW

File: website/content/posts/introducing-wash.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+In fact, Wash is not just a centralization of basic and fundamental commands. It is a shell. It is a shell layered on top of an existing shell like Bash or ZSH. This means that:
+
+* “Everything is a file”. Thus, interacting with a specific kind of thing like a Docker container is as easy as cd’ing into its containing “directory” and running a bunch of commands on the things in that “directory”. For example, something like cd docker/containers lets you follow-up with commands like ls (list all containers), wexec foo uname (execute the uname command on the foo container), or tail foo/fs/var/log/messages (tail the foo container’s /var/log/messages file for updates).
+
+* You can tab-complete and glob stuff (if your shell supports those features). For example, assuming you’re in the docker/containers directory, typing in ls f then tab will work just the way you’d expect it to work if you were in a standard directory like /var/log (where f would be a regular file or directory). Similarly, tail */fs/var/log/messages will tail every container’s /var/log/messages file for updates.
```
Comment (ekinanp) [inline website/content/posts/introducing-wash.md]: No footnote?

Comment (MikaelSmith) [reply website/content/posts/introducing-wash.md]: Haven't figured out the formatting yet. Also not sure if it's necessary.

Comment (MikaelSmith) [reply website/content/posts/introducing-wash.md]: Updated.

Comment (ekinanp) [reply website/content/posts/introducing-wash.md]: Sweet. Small error in `tail find -k ‘aws/*ec2*instance’ -o -k ‘gcp/*compute*instance’ | sed s:$:/fs/var/log/messages:g`. This should be
```
tail `find -k 'aws/*ec2*instance' -o -k 'gcp/*compute*instance' | sed s:$:/fs/var/log/messages:g`
```

File: website/content/posts/introducing-wash.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+* You no longer have to install a bunch of different tools. All you need to install is the Wash binary.
+
+* You no longer have to learn different DSLs for filtering stuff. All you need to learn is find’s expression syntax and its individual primaries. Once you do that, you can filter on almost any conceivable property of your specific thing.
+
+In fact, Wash is not just a centralization of basic and fundamental commands. It is a shell. It is a shell layered on top of an existing shell like Bash or ZSH. This means that:
+
+* “Everything is a file”. Thus, interacting with a specific kind of thing like a Docker container is as easy as cd’ing into its containing “directory” and running a bunch of commands on the things in that “directory”. For example, something like cd docker/containers lets you follow-up with commands like ls (list all containers), wexec foo uname (execute the uname command on the foo container), or tail foo/fs/var/log/messages (tail the foo container’s /var/log/messages file for updates).
```
Comment (ekinanp) [inline website/content/posts/introducing-wash.md]: Hmm, some command invocations are missing some highlights (i.e. the backticks). Was that intentional?

File: website/content/posts/introducing-wash.md (hunk 1 above)
Comment (MikaelSmith) [reply website/content/posts/introducing-wash.md]: Updated.

File: website/content/posts/introducing-wash.md (hunk 2 above)
Comment (ekinanp) [reply website/content/posts/introducing-wash.md]: Also removed the quotes from "Everything is a file." Reads a bit better that way.

Comment (MikaelSmith) [reply website/content/posts/introducing-wash.md]: Probably not. They don't copy over from google docs.

**Comment by MikaelSmith** [comment]
👍

**Review by ekinanp — APPROVED** [review]

**Pull request merged**